UnitTestingApp.js

```diff
@@ -8,19 +8,27 @@
  * Class for running unit tests
  */
 
- let UnitTestingApp = (function () {
+let UnitTestingApp = (function () {
 
+  _nTests = 0;      // Total number of tests executed
+  _nFailTests = 0;  // Total test failed
+  _nPassTests = 0;  // Total test passed
+  _levelInfo = 1;   // Level of information to show in the output (0-summary, 1-trace and test result information)
   const _enabled = new WeakMap();
   const _runningInGas = new WeakMap();
 
   class UnitTestingApp {
     constructor() {
       if (UnitTestingApp.instance) return UnitTestingApp.instance;
-      
+
       _enabled.set(this, false);
       _runningInGas.set(this, false);
+      this._nTests = 0;
+      this._nFailTests = 0;
+      this._nPassTests = 0;
+      this._levelInfo = 1;
```

Comment:
I thought you said you switched to the `WeakMap`?

Reply:
yes, but because I had other question/ comments, I haven't updated the code.